### PR TITLE
PUB-1591 - Deploy CRON timers to other environments

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1591

### Change description ###

Updated CRON timers deployment to other environments

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by joshblackmoor — APPROVED** [review]

**Review by TomKimber — APPROVED** [review]

File: apps/pip/account-management-inactive-verification-cron/demo/00.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  releaseName: pip-account-management-inactive-verify-cron
+  values:
+    job:
+      schedule: "01 08 * * 0,2,4,6"
+      environment:
+        DATA_MANAGEMENT_URL: https://pip-data-management.demo.platform.hmcts.net
+        ACCOUNT_MANAGEMENT_URL: https://pip-account-management.demo.platform.hmcts.net
+        TRIGGER_TYPE: ACCOUNT_INACTIVE_VERIFICATION
+    global:
```
Comment (adusumillipraveen) [inline apps/pip/account-management-inactive-verification-cron/demo/00.yaml:14]: You might not need these to my knowledge

File: apps/pip/dev/01/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: pip
+
+resources:
+  - ../base
+
+patchesStrategicMerge:
```
Comment (adusumillipraveen) [inline apps/pip/dev/01/kustomization.yaml:8]:
```suggestion
```

File: apps/pip/ithc/00/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: pip
+
+resources:
+  - ../base
+
+patchesStrategicMerge:
```
Comment (adusumillipraveen) [inline apps/pip/ithc/00/kustomization.yaml:8]:
```suggestion
```

File: apps/pip/ithc/01/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: pip
+
+resources:
+  - ../base
+
+patchesStrategicMerge:
```
Comment (adusumillipraveen) [inline apps/pip/ithc/01/kustomization.yaml:8]:
```suggestion
```

File: apps/pip/prod/00/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: pip
+
+resources:
+  - ../base
+
+patchesStrategicMerge:
```
Comment (adusumillipraveen) [inline apps/pip/prod/00/kustomization.yaml:8]:
```suggestion
```

File: apps/pip/prod/01/kustomization.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: pip
+
+resources:
+  - ../base
+
+patchesStrategicMerge:
```
Comment (adusumillipraveen) [inline apps/pip/prod/01/kustomization.yaml:8]:
```suggestion
```

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
minor comments, I don't think u need global flags. that should all be set from https://github.com/hmcts/sds-flux-config/blob/master/apps/base/kustomize.yaml#L31

File: apps/pip/prod/00/kustomization.yaml (hunk 5 above)
Comment (ChrisS1512) [reply apps/pip/prod/00/kustomization.yaml]: Removed :)

File: apps/pip/prod/01/kustomization.yaml (hunk 6 above)
Comment (ChrisS1512) [reply apps/pip/prod/01/kustomization.yaml]: Removed :)

File: apps/pip/ithc/01/kustomization.yaml (hunk 4 above)
Comment (ChrisS1512) [reply apps/pip/ithc/01/kustomization.yaml]: Removed :)

File: apps/pip/ithc/00/kustomization.yaml (hunk 3 above)
Comment (ChrisS1512) [reply apps/pip/ithc/00/kustomization.yaml]: Removed :)

File: apps/pip/dev/01/kustomization.yaml (hunk 2 above)
Comment (ChrisS1512) [reply apps/pip/dev/01/kustomization.yaml]: Removed :)

File: apps/pip/account-management-inactive-verification-cron/demo/00.yaml (hunk 1 above)
Comment (ChrisS1512) [reply apps/pip/account-management-inactive-verification-cron/demo/00.yaml:14]: Thanks. I've removed these from 'test' (except for jobKind: CronJob) to try that out. If it works in that env once deployed I'll go through the other env's and remove them too :)

**Review by adusumillipraveen — APPROVED** [review]

**Review by furnivall — APPROVED** [review]

**Pull request merged**